src/html.rs

```diff
@@ -235,11 +235,16 @@ fn get_base(rel_path: &Path) -> String {
 fn get_dirs_result(global: Arc<Mutex<HtmlGlobalStats>>, rel_path: &Path, stats: &HtmlStats) {
     let parent = rel_path.parent().unwrap().to_str().unwrap().to_string();
     let file_name = rel_path.file_name().unwrap().to_str().unwrap().to_string();
+    let mut global = global.lock().unwrap();
     let fs = HtmlFileStats {
         stats: stats.clone(),
+        abs_prefix: global
+            .abs_prefix
+            .as_ref()
+            .map(|p| format!("{}{}/", p, parent)),
```

Comment:
it'd be ideal, here and elsewhere, to join the abs path and the path with proper formatting and not just as string (otherwise we must ask users to add "/" at the end)

Reply:
Ahh yes, good point.  I've updated it to use `PathBuf` instead of string concatenation.